packages/ingest-mongodb-public/environments/production.yml

```diff
@@ -31,6 +31,34 @@ cronJobs:
         cpu: 4
         memory: 5Gi
     backoffLimit: 3
+  - name: ingest-pages-with-links
```

Comment:
This env update feels out of place - just marking here for you to merge or double check that it won't be an issue on merge

Reply:
oh definitely. good catch. this was for other PR i was working on at similar time. must have gotten caught up in gitfoo.

will remove from PR